index.ts

```diff
@@ -259,20 +259,60 @@ async function splitNeuron(neuronId: bigint, amount: bigint) {
   ok();
 }
 
-async function spawnNeuron(neuronId: string, controller?: Principal) {
+async function spawnNeuron(neuronId: string, controller?: Principal, percentage?: number) {
   const identity = await getLedgerIdentity();
+  // Percentage is only supported with Candid transaction which was added in version 2.2.0
+  if (percentage !== undefined) {
+    await assertLedgerVersion({ identity, minVersion: "2.2.0" });
```

Comment:
not in this PR but maybe you can extract some constants for these versions? maybe useful for search purposes

Reply:
I'll add it in this PR, good point.